recipes/user_install.rb

```diff
@@ -26,6 +26,27 @@
   # if user hash is not a hash (i.e. set to true), init an empty Hash
   opts = Hash.new if opts == true
 
+  ruby_block 'Conditionally add RVM gpg key' do
```

Comment:
Perhaps try changing this to: `ruby_block "Conditionally add RVM gpg key for #{user}" do`